src/js/components/modals/EditAppModalComponent.jsx

```diff
@@ -73,7 +73,8 @@ var EditAppModalComponent = React.createClass({
 
   onRequestAppVersionError: function () {
     var props = this.props;
-    DialogActions.alert(`Could not fetch app '${props.appId}' version for editing: ${props.appVersion}`);
+    DialogActions.alert(`Could not fetch app '${props.appId}' version`
```

Comment:
@philipnrmn but here you kept it at the end: https://github.com/mesosphere/marathon-ui/pull/184/files#diff-ee58f29f51a4e5443773b0fed24eb749R86